src/components/custom-field-input-currency/custom-field-input-currency.jsx

```diff
@@ -73,6 +74,12 @@ export default function CustomFieldInputCurrency(props) {
     }
   });
 
+  useImperativeHandle(ref, () => ({
+    value: () => {
```

Comment:
We should keep this as a getter for a static value instead of a function:

```suggestion
    value: [parseInt(componentRef.current.value().replace(/\D/g, ''), 10), props.currencyCode],
```

Reply:
@juanca I'm not sure exactly why, but without this in `function` form, the fetched value does not seem to change.